Deeploy/Targets/Generic/Parsers.py

```
@@ -2697,6 +2697,51 @@ def parseNodeCtxt(self,
+        n_outputs = len(node.outputs)
+        num_tensors = (n_inputs - 2) // 4
+        valid_inputs = n_inputs >= 6 and (n_inputs - 2) % 4 == 0
+        valid_outputs = n_outputs >= 1 and n_outputs == num_tensors
+        valid_attrs = all(a in node.attrs for a in ['alpha', 'beta', 'epsilon', 'norm_coefficient', 'norm_coefficient_post'])
+
+        return all([valid_inputs, valid_outputs, valid_attrs])
+
+    def parseNodeCtxt(self,
+                      ctxt: NetworkContext,
+                      node: gs.Node,
+                      channels_first: bool = True) -> Tuple[NetworkContext, bool]:
```

Ruff (0.15.2)

[warning] Unused method argument: `channels_first`

(ARG002)